src/Content/Nav.php

```diff
@@ -245,7 +245,7 @@ private static function getInfo(App $a): array
 
 		$gdirpath = 'directory';
 
-		if (strlen(DI::config()->get('system', 'singleuser'))) {
+		if (empty(DI::config()->get('system', 'singleuser'))) {
```

Comment:
`empty` has the opposite logical meaning as `strlen` in this case. Please fix using either suggestion:
```suggestion
		if (!empty(DI::config()->get('system', 'singleuser'))) {
```
```suggestion
		if (DI::config()->get('system', 'singleuser') ?? '') {
```